File: lib/utils/formatters.dart
class PriceFormatter {
  static String formatPrice(double price) {
    return '\$${price.toStringAsFixed(2)}';
  }

  static String formatChange(double change) {
    final prefix = change >= 0 ? '+' : '';
    return '$prefix${change.toStringAsFixed(2)}';
  }

  static String formatPercentage(double percentage) {
    final prefix = percentage >= 0 ? '+' : '';
    return '$prefix${percentage.toStringAsFixed(2)}%';
  }
}

File: src/utils/formatters.ts
export const formatPrice = (price: number): string => {
  return `$${price.toFixed(2)}`;
};

export const formatChange = (change: number): string => {
  const prefix = change >= 0 ? '+' : '';
  return `${prefix}${change.toFixed(2)}`;
};

export const formatPercentage = (percentage: number): string => {
  const prefix = percentage >= 0 ? '+' : '';
  return `${prefix}${percentage.toFixed(2)}%`;
};